tests/step-011.sh

```diff
@@ -2,13 +2,25 @@
 # Test:
 #   Direct template execution: test pre-commit hooks
 
+mkdir -p ~/.githooks/release && cp /var/lib/githooks/*.sh ~/.githooks/release || exit 1
```

Comment:
Can we not do this in the test setup code rather than repeated in each test?

Reply:
Actually, shouldn't the install take care of this?

Reply:
This **only** and really only for *direct* template execution. We place them there, to counteract coverage.

Reply:
Can't be done globall for all tests. 
Only for them which use *direct* template execution.

Reply:
Ah OK. 👍